vtex/admin-promotions
# Fix/ch27830/promotions input currency missing intl prop

#### What is the purpose of this pull request?
Add missing `intl` prop to `PromotionsInputCurrency` component in `ShippingForm`.

#### What problem is this solving?
This missing prop was causing the form page to explode when this option was selected.

#### How should this be manually tested?
1. Go to https://eric--storecomponents.myvtex.com/admin/promotions/new/
2. Select the **Shipping** card in the effects section
3. Select **Maximum value** option
4. See that it does not explode like in https://storecomponents.myvtex.com/admin/promotions/new/

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/5971264/70553070-3ac13000-1b59-11ea-9d36-9956578a8b58.png)
